autogpt_platform/backend/backend/blocks/text.py

```diff
@@ -2,25 +2,27 @@
 from typing import Any
 
 from jinja2 import BaseLoader, Environment
-from pydantic import Field
 
 from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import SchemaField
 from backend.util import json
 
 jinja = Environment(loader=BaseLoader())
 
 
 class MatchTextPatternBlock(Block):
     class Input(BlockSchema):
-        text: Any = Field(description="Text to match")
-        match: str = Field(description="Pattern (Regex) to match")
-        data: Any = Field(description="Data to be forwarded to output")
-        case_sensitive: bool = Field(description="Case sensitive match", default=True)
-        dot_all: bool = Field(description="Dot matches all", default=True)
+        text: Any = SchemaField(description="Text to match")
```

Comment:
nit: does this actually work on any

Reply:
It behaves the same as the original Field